frontend/src/hooks/useStatistics.ts

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from "react-query";
+
+import { ApiService } from "src/services/ApiService";
+
+import { UserStatisticsRes } from "~shared/types/api";
+
+export const useGetUserStatistics = () => {
+  const getUserStatistics = async () => {
+    const { data } = await ApiService.get<UserStatisticsRes | undefined>(
+      `/user/statistics`,
+    );
+    return data;
+  };
+  const { data: statistics, isLoading: isGetUserStatisticsLoading } = useQuery(
+    ["STATISTICS"],
```

Comment:
Update this to use the `QUERY_KEY` object instead for consistency after updating with main.